docs/includes/file-tutorial-excel-custom-functions.md

```diff
@@ -53,12 +53,37 @@ You’ll begin this tutorial by using the Yo Office generator to create the file
     cd stock-ticker
     ```
 
-3. Start the local web server.
+3. Trust the self-signed certificate that is needed to run this project.
+
+    ```bash
+    cd certs
+    start ca.crt
+    ```
+
+    This will bring up a pop-up window. Follow the below directions to trust the self-signed certificate:
```

Comment:
Sure - I was trying to avoid the whole "what I have to follow another link" problem but I agree that it is better to have a link for mac folks, didn't think about that.